alchemist/src/orchid/resources/wiki/usage/indoor.md

```diff
@@ -4,4 +4,16 @@ title: "Simulation of indoor environments"
 
 ---
 
+In order to load the map of the indoor environment, you can use the {{ anchor('ImageEnvironment') }}.
 
+ImageEnvironment loads the map as raster image from file, interpreting the black pixels as obstacles
+(areas that cannot be accessed by nodes like wall). 
```

Comment:
"wall-like areas not accessible to nodes"

Reply:
resolved